translations/af.json

```diff
@@ -1,6 +1,7 @@
 {
 	"LOADING": "Besig om te laai …",
 
```

Comment:
Is it intentional that some languages are missing DAYBEFOREYESTERDAY?

Reply:
There is no useful translation for "DAYBEFOREYESTERDAY" and "DAYAFTERTOMORROW" in some languages. The calendar module takes care of these exceptions for "DAYAFTERTOMORROW" already.
I adapted the mechanism to "DAYBEFOREYESTERDAY"

Reply:
Ah okay, thanks for the explanation!

I think this PR is a useful addition to the standard module 🙂